home-work-app/src/app/pages/home/home.page.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Banner } from './components/banner.cmpt';
+
+export class HomePage extends React.PureComponent {
```

Comment:
It can be just a functional component for now
And HomePage include not only header, search input and footer, it also contains list of movies

Reply:
Discussed